frontend/src/components/ProductList.js

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import styled from 'styled-components';
 import ProductCard from './ProductCard';
-
+import axios from 'axios';
 
 const ProductList = () => {
+    
+    const [ products, setProducts] = useState([])
+
+    useEffect(
+        () => {
+            axios.get('https://my-json-server.typicode.com/menezesmario/fake_api_products/products')
+            .then(response => {
+                const dataProducts = response.data
+                setProducts(dataProducts)
+            });
+            
+        }, [])     
+   
     return (
+
         <MainContent>
+            
         <TitleList>Os mais vendidos</TitleList>
-        <ProductsContainer>   
-                        <ProductCard/>
-            <ProductCard/>
-            <ProductCard/>
-            <ProductCard/>
-            <ProductCard/>
-            <ProductCard/>
+        <ProductsContainer>        
+            {products.map( 
+                product => <div key={product.id}>
+                <ProductCard 
+                title={product.title}
+                price={product.price}
+                oldPrice={product.oldPrice}
+                description={product.description}
+                imageUrl={product.imageUrl}
+                categoryId={product.categoryId}                
+                />
+                </div>                
+            )}  
+        
```

Comment:
Eu trocaria essa `div` por volta do `ProductCard` para um `li`, afinal, você está listando alguns itens então faz sentido usar `ul` + `li`.

```suggestion
            {products.map(product => (
                <li key={product.id}>
                    <ProductCard 
                        title={product.title}
                        price={product.price}
                        oldPrice={product.oldPrice}
                        description={product.description}
                        imageUrl={product.imageUrl}
                        categoryId={product.categoryId}
                    />
                </li>
            ))}
```

Com isso, vale trocar o `ProductsContainer` para `ul`, e talvez adicionar um estilo para remover o esitlo de lista padrão do HTML:
```jsx
const ProductsContainer = styled.ul`
    list-style: none;
...
```
`ol` -> Lista ordenada (ordered list)
`ul` -> Lista não ordenada (unordered list)
`li` -> Item da lista (list item)